patches/minecraft/net/minecraft/enchantment/Enchantment.java.patch

```diff
@@ -12,7 +12,18 @@
      private final EntityEquipmentSlot[] field_185263_a;
      private final Enchantment.Rarity field_77333_a;
      @Nullable
-@@ -134,7 +134,7 @@
+@@ -94,6 +94,10 @@
+         return 0;
+     }
+ 
++    /**
++     * Consider using Forge's entity sensitive version
++     */
```

Comment:
Do __*not*__ add Javadoc to Vanilla methods. Their Javadoc is part of the MCP mappings and should not be modified by Forge. The convention is to just stick a normal comment after `@Deprecated`.